main.ts

```diff
@@ -2,11 +2,18 @@ import Router, { RoutesList } from './lib/router.ts';
 import Server from './lib/server.ts';
 import { RouteOptions, RouteHandler, ServerOptions } from './lib/types.ts';
 
-export default {
+export * as types from './lib/types.ts'
+
+export { default as Router } from './lib/router.ts';
+export { default as Server } from './lib/server.ts';
+
+export const Main = {
```

Comment:
This implies `import { Main } from 'https://deno.land/x/pogo/main.ts'` is a fine pattern, but I would find that very unclear. How about instead of exporting a `Main` namespace, instead we provide named exports for the `server()` and `router()` factories and then also `export default` an object with them. I think the `deno doc` output should still be good, but correct me if I'm wrong.

Reply:
👍